src/app/pages/fund/components/fiat-providers.utils.ts

```diff
@@ -67,8 +67,8 @@ function makeTransakUrl(address: string) {
   )}&defaultPaymentMethod=credit_debit_card&walletAddress=${address}`;
 }
 
-function makeGenericExternalUrl(address: string, provider: string) {
-  return `https://hiro.so/wallet-faq/exchange-purchase?provider=${provider}&address=${address}`;
+function makeGenericExternalUrl(address?: string, provider?: string) {
```

Comment:
@fbwoolf Any particular reason why it's optional now ?

Reply:
Big +1

If this method is sometime invoked with undefined values, that's a concern of the code _ivoking_ the function, not the function itself.

Reply:
That is a mistake, thanks for catching. I was testing something and forgot to remove.